apps/session-manager/src/services/redis-session-storage.ts

```diff
@@ -734,6 +735,71 @@ export const set =
     );
   };
 
+/**
+ * Delete all tokens related to the user
+ * @param user
+ * @returns
+ */
+export const deleteUser: (
+  user: User,
+) => (deps: RedisRepo.RedisRepositoryDeps) => TE.TaskEither<Error, boolean> =
```

Comment:
alternative format. Also it seems the right part of the TaskEither is always `true` instead of `boolean`
```suggestion
export const deleteUser: (
  user: User,
) => RTE.ReaderTaskEither<RedisRepo.RedisRepositoryDeps, Error, true> =
```